joomla/joomla-cms
# [4.0] Align dashboard module titles in files 4.0.0-2019-07-13.sql with joomla.sql files

Pull Request for Issue # .

### Summary of Changes

With Pull Request (PR) #26047 , titles of dashboard modules have been changed to more meaningful "... Dashboard" instead of "... Submenu". A big thank you to the author for that change. Previous titles were really confusing.

Since all these modules have been added with the new backend template, i.e. with update SQL script "4.0.0-2019-07-13.sql", it makes sense to do the same changes in this update SQL script so a 4.0 beta or rc or final updated from a 3.10 will have the same module title as it would be after a new installation for these particular modules.

This is what this PR does.

**General remark on changing the SQL update script**

Since we are not in beta yet and so don't have to support updates from 4.0-Alpha-x to 4.0-Alpha-y or between nightly builds, we can change the existing 4.0 update scripts (but of course not pre-4.0 scripts). Later when in beta this will not be allowed anymore because we have to support updating from Beta-x to Beta-y (with y > x of course), so now is a good time to fix it.

### Testing Instructions

Code review: Verify for each MySQL and PostgreSQL that the changed made by this PR here in file "4.0.0-2019-07-13.sql" are identical with the changes for inserting new modules made by PR #26047 in file "joomla.sql" for the same dabatase type.

### Expected result

SQL update scripts "4.0.0-2019-07-13.sql" and corresponding "joomla.sql" are consistent regarding the titles of dashboard modules inserted by the new backend template, which are 'System Dashboard', 'Content Dashboard', 'Menus Dashboard', 'Components Dashboard', 'Users Dashboard' and 'Help Dashboard'.

### Actual result

In SQL update scripts "4.0.0-2019-07-13.sql" those modules mentioned in section "Expected result" are named "... Submenu" and not "... Dashboard", like it should be and like it is in corresponding file "joomla.sql".

### Documentation Changes Required

None I guess.
